### ISSUE TITLE
Bug: Textarea and other common words replaced

### ISSUE THREAD

**Issue by pacifier007** [JPeer264/node-rename-css-selectors#84]
Version info: 
```
    "rcs-core": "^3.7.0",
    "rename-css-selectors": "^4.1.0",
```

Example CSS:

```css
textarea { color: red; }
.text { color: green; }
```

Result:

```
tarea { color: red; }
.t { color: green; }
```

Simple normal JS:
```
	rcs.process.auto('css/amp.css', {ignoreCssVariables: true})
		.then(() => rcs.mapping.generate('./inc', {fileName: 'map', overwrite: true}))
```

My CSS file was very large with many replacements gone awry, but I created a simpler case to present the bug. 

Is the parser/regex broken somewhere. That's really odd that it's doing a simple replace such as this. Not sure what's going on. This used to work fine in 3.x.

**Comment by JPeer264** [comment]
Hey thanks for your issue. 

Oofs, that is definitely a bug. I will look into it rn.

**Comment by JPeer264** [comment]
should be fixed in `rcs-core@3.7.1`

**Comment by pacifier007** [comment]
Perfect. That was an amazingly quick fix. Many thanks.
